gitlabform/gitlabform/processors/project/merge_requests_processor.py

```diff
@@ -1,6 +1,7 @@
 import logging
 
 import cli_ui
+from distutils.version import LooseVersion
```

Comment:
Have you considered using `packaging.version.parse()` instead? It seems to be the preferred tool for processing version in Python currently, see https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python .

We also already use it it https://github.com/egnyte/gitlabform/blob/main/dev/check_release_type.py#L3 .

Reply:
That one gives me strange results due to the `-ee` suffix in the GitLab. Maybe sanitize it first with something like 
```
self.version = re.match('\d+\.\d+\.\d+', version['version']).group(0)
```

and then use `packaging`?

```
>>> version.parse('13.11.3-ee') < version.parse('13.11')
True  # weird
```

Reply:
I checked that and '13.11.3-ee' is parsed as `LegacyVersion` (because it's not PEP440-compliant) and `13.11` as `Version` and as [noted here](https://packaging.pypa.io/en/latest/version.html#packaging.version.LegacyVersion): 
> LegacyVersion instances are always ordered lower than Version instances.

I would not code it like that, it's a trap for the users IMHO.

In this case I would leave your code as it is then. :)